versions/3.1.1.md

```diff
@@ -2757,43 +2705,44 @@ Where JSON Schema indicates that behavior is defined by the application (e.g. fo
 
 ##### Properties
 
-The OpenAPI Schema Object [dialect](https://tools.ietf.org/html/draft-bhutton-json-schema-00#section-4.3.3) is defined as requiring the [OAS base vocabulary](#baseVocabulary), in addition to the vocabularies as specified in the JSON Schema draft 2020-12 [general purpose meta-schema](https://tools.ietf.org/html/draft-bhutton-json-schema-00#section-8).
+The OpenAPI Schema Object [dialect](https://tools.ietf.org/html/draft-bhutton-json-schema-00#section-4.3.3) is defined as requiring the [OAS base vocabulary](#oas-base-vocabulary), in addition to the vocabularies as specified in the JSON Schema draft 2020-12 [general purpose meta-schema](https://tools.ietf.org/html/draft-bhutton-json-schema-00#section-8).
```

Comment:
This pull request is not ready to merge because there's a broken link here - I am not sure where "OAS base vocabulary" should link to - and if it's one of these same-named sections, I'm not sure how to express that.

Reply:
i believe it should be `#fixed-fields-19`

that's where those OAS specific keywords are defined.

Reply:
I suggest re-inserting that link target into the generic headline

Reply:
Added the anchor link